include/gmapping/grid/array2d.h

```diff
@@ -7,6 +7,10 @@
 
 #include <iostream>
 
+#ifdef _WIN32
+#define __PRETTY_FUNCTION__ __FILE__":" << __LINE__
```

Comment:
Shouldn't `__PRETTY_FUNCTION__` include the function name? Maybe the fallback should be `typeid(func).name()`?